src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -523,7 +524,7 @@ public static PublishPackageViewModel FromLocalPackage(DynamoViewModel dynamoVie
                 Description = l.Description,
                 Keywords = l.Keywords != null ? String.Join(" ", l.Keywords) : "",
                 CustomNodeDefinitions = defs,
-                Assemblies = l.LoadedAssemblies.ToList(),
+                Assemblies = new List<PackageAssembly>(),
```

Comment:
What if the package already has loaded assemblies?  This would remove them from the package.